flight/Modules/Actuator/actuator.c

```diff
@@ -691,7 +589,7 @@ static bool set_channel(uint8_t mixer_channel, float value, const ActuatorSettin
                             }
 			}
 			PIOS_Servo_Set(mixer_channel,
-							buzzOn?actuatorSettings->ChannelMax[mixer_channel]:actuatorSettings->ChannelMin[mixer_channel]);
+							buzzOn?actuatorSettings.ChannelMax[mixer_channel]:actuatorSettings.ChannelMin[mixer_channel]);
```

Comment:
Agreed. I think historically the general rule has been to fix style problems whenever the nearby functions are changed. But this process is taking so long, and we're not using astyle for new functions/files, so I think the problem is getting worse faster than it's getting better.